ghost/core/test/unit/frontend/helpers/ghost_head.test.js

```diff
@@ -32,29 +64,33 @@ describe('{{ghost_head}} helper', function () {
             meta_description: 'tag meta description',
             name: 'tagtitle',
             meta_title: 'tag meta title',
-            feature_image: '/content/images/tag-image.png'
+            feature_image: '/content/images/tag-image.png',
+            updated_at: new Date(0)
```

Comment:
Curious why this has been added, `createTag` should already handle setting a date?

Reply:
The issue was that the default value is `new Date()`, and some of the structured data in the head would change every time for some tests with structured data included. The string based snapshots sadly don't support some kind of `anyDate` matcher.